src/components/ToggleButton.css

```diff
@@ -1,35 +1,52 @@
 .onoffswitch {
-    position: relative; width: 40px;
-    margin-top: 5px;
+    position: relative; width: 43px;
     -webkit-user-select:none; -moz-user-select:none; -ms-user-select: none;
 }
 .onoffswitch-checkbox {
     display: none;
 }
 .onoffswitch-label {
     display: block; overflow: hidden; cursor: pointer;
-    height: 13px; padding: 0; line-height: 13px;
-    border: 0px solid #FFFFFF; border-radius: 20px;
-    background-color: #42A5F5;
+    border: 2px solid #999999; border-radius: 10px;
 }
-.onoffswitch-label:before {
+.onoffswitch-inner {
+    display: block; width: 200%; margin-left: -100%;
+    transition: margin 0.3s ease-in 0s;
+}
+.onoffswitch-inner:before, .onoffswitch-inner:after {
```

Comment:
What's the planned browser support for this app.

Single colon notation for pseudo elements is only legacy IE stuff afaik. :)